src/test/resources/application-default.yml

```diff
@@ -0,0 +1,20 @@
+spring:
+  jpa:
+    database: H2
+    properties:
+      dialect: org.hibernate.dialect.H2Dialect
+      hibernate.format_sql: true
```

Comment:
some projects have `hibernate.generate_statistics: true` here as well... don't know if it's relevant... but just saying there is this difference

Reply:
yes we have to add it for all projects
it's use to make some optimisations by analyzing the metrics

Reply:
OK